tasks/setup-RedHat.yml

```diff
@@ -5,3 +5,10 @@
     state: "{{ php_packages_state }}"
     enablerepo: "{{ php_enablerepo | default(omit, true) }}"
   notify: restart webserver
+
+- name: Add and enable Remi PHP 7.3.x YUM repo
+  yum_repository:
+    name: remi-php73
+    description: Remi PHP 7.3.x YUM repo
+    baseurl: "https://rpms.remirepo.net/enterprise/remi-release-{{ ansible_distribution_major_version }}.rpm"
+    enabled: yes
```

Comment:
Not sure we want to hard-code the repo name here given it's set within the `php_enablerepo` variable for use in e.g. the previous task.  How about the following? 

```yaml
- name: Enable relevant repos so we get upgrades
  yum_repository:
    name: "{{ item }}"
    enabled: yes
  with_items: "{{ php_eenablerepo.split(',') }}"
```

Reply:
With the variable set in `defaults/main.yml`?

Reply:
Just testing with vagrant...

Reply:
What about:
```yaml
- name: Enable PHP repo so we get upgrades
  yum_repository:
    name: "{{ php_enablerepo }}"
    baseurl: "{{ remi_repo_url }}"
    gpgkey: "{{ remi_repo_gpg_key_url }}"
    enabled: yes
```
I think there's only one repo - do we need to allow for more? If so we need urls etc or I'm getting confused...

Reply:
Looks like that would also work if php_enablerepo is just the name of one repo (rather than being a comma-delimited list as it is here: https://github.com/geerlingguy/ansible-role-php/issues/134).